plugins/inputs/zipkin/codec/jsonV1/jsonV1_test.go

```diff
@@ -870,13 +870,13 @@ func TestTraceIDFromString(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := TraceIDFromString(tt.s)
+			got, err := traceIDFromString(tt.s)
 			if (err != nil) != tt.wantErr {
-				t.Errorf("TraceIDFromString() error = %v, wantErr %v", err, tt.wantErr)
+				t.Errorf("traceIDFromString() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
 			if got != tt.want {
-				t.Errorf("TraceIDFromString() = %v, want %v", got, tt.want)
+				t.Errorf("traceIDFromString() = %v, want %v", got, tt.want)
```

Comment:
As you already touching this, do you think it's worth directly migrating those test to `require`? Same for the places below.

Reply:
Sure, changed in whole `zipkin` package